compiler-core/src/type_/error.rs

```diff
@@ -1681,3 +1686,25 @@ pub fn check_javascript_int_safety(int_value: &BigInt, location: SrcSpan, proble
         problems.warning(Warning::JavaScriptIntUnsafe { location });
     }
 }
+
+/// When targeting Erlang, adds an error if the given Float value is outside the range
+/// -1.7976931348623157e308 to 1.7976931348623157e308 which is the allowed range for
+/// Erlang's floating point numbers
```

Comment:
Could we link to there these numbers were taken from in the Erlang docs please 🙏

Reply:
I actually can't find them anywhere in the Erlang docs. I got them from [here](https://github.com/elixir-lang/elixir/blob/v1.18.1/lib/elixir/lib/float.ex).